src/trigger.py

```diff
@@ -38,6 +38,9 @@ def _run_trigger(args, build_config, db):
     else:
         node_list = get_node_by_commit_id(
             kernelci.build.get_branch_head(build_config))
+        if len(node_list) != 0:
+            print(f"Node exists with the latest git commit")
+            return 0
```

Comment:
I don't think there's any expected return value from this method, so maybe just `return` is better.  That would actually return `None`.